samples/drivers/lora/send/boards/96b_wistrio.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SPI_1=y
```

Comment:
Can't we put this directly in board's .defconfig under `if LORA`?

Reply:
IIUC, it is the duty of sample applications to enable the drivers needed for the functionality and board defconfig should only enable the bare minimum ones needed for basic usage. Am I missing something?

Reply:
This is true, but doesn't prevent to do some conditional configuration in the board itself when we know it'll be always valid (part of the board BSP).
So in the board something like:
```
if LORA
config SPI_1
     default  y
```
would be always valid and do not enable stuff when not required.
It is still application responsibility to enable `LORA` and `SPI_1` is not enabled when LORA is not required (Non-LORA application for instance).